app/controllers/collections_controller.rb

```diff
@@ -41,19 +41,57 @@ def create
   # rubocop:enable Metrics/MethodLength
   # rubocop:enable Metrics/AbcSize
 
+  # this is a special route meant to edit a first draft (non deposited) version of a collection
+  # so that we can show the complete form instead of the split details/settings form
+  def edit_first_draft
```

Comment:
Can you move these to a separate controller (e.g. FirstDraftCollectionController) or can we move the "Collection settings" related stuff to a different controller?  Having more than one form per controller seems more complicated than it ought to be

Reply:
i can create a new controller for these two new actions like you suggest